modules/qna/src/backend/utils.ts

```diff
@@ -61,22 +61,24 @@ export const getQnaEntryPayloads = async (
     )
   }
 
-  for (const contentAnswer of qnaEntry.contentAnswers[lang]) {
-    renderer = `#${contentAnswer.contentType}`
-    const contentArgs = {
-      ...args,
-      ...contentAnswer,
-      typing: payloads.length === 0
+  if (qnaEntry.contentAnswers) {
```

Comment:
You could invert that condition to save one level of indentation (return payloads early)